dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/ConsoleMonitorThread.java

```diff
@@ -60,8 +60,13 @@ public void run() {
             try {
                 int state = server.getServerState();
                 WebSphereRuntime wsRuntime = getWebSphereRuntime();
-                // server process is running, but we aren't tracking the state yet so need to add the process listener to listen for console messages
-                if (wsRuntime != null && wsServer.getUserDirectory() != null && wsRuntime.isServerStarted(wsServer.getServerInfo(), null)) {
+                int serverStatus = -2;
+                if (wsRuntime != null && wsServer.getUserDirectory() != null) {
+                    serverStatus = wsRuntime.getServerStatus(wsServer.getServerInfo(), 20f, new NullProgressMonitor());
+                }
+                if (serverStatus == 0) {
```

Comment:
Same here, magic number `0` is nowhere explained.

Reply:
I added a comment